uk_bin_collection/uk_bin_collection/councils/EastSuffolkCouncil.py

```
@@ -87,16 +86,34 @@ def parse_data(self, page: str, **kwargs) -> dict:
+
+                    # Strip " - standard bin"
+                    lower_bin_type = bin_type.lower()
+                    if " - standard bin" in lower_bin_type:
+                        idx = lower_bin_type.find(" - standard bin")
+                        bin_type = bin_type[:idx]
+
+                    bin_type = bin_type.strip().title()
+
+                    collection_date = datetime.strptime(
+                        collection_date, "%d/%m/%Y"
+                    ).strftime(date_format)
```

Ruff (0.15.7)

[error] `date_format` may be undefined, or defined from star imports

(F405)